mimicpy/parsers/top_reader.py

```diff
@@ -177,9 +182,14 @@ def _parseatoms(self, txt, file_name):
                 mass_int = int(mass)
                 
                 if mass_int <= 0:
-                    raise ParserError(file=file_name, ftype="topolgy", \
-                        extra=(f"Cannot guess atomic symbol for atom with name {name} and type {_type} in residue {res} as mass"
-                                             "information is not available from the force field"))
+                    if name in element_names:
```

Comment:
Since MiMiC needs the atom information anyway, I would suggest to drop the element guessing completely.

Reply:
No. I think it's convenient to guess the atomic numbers. I also want to give an option of editing the topology file to include these atomic symbols for non standard atoms. This way, mimicpy can do everything for you: give you the cpmd file, ndx file, the correct topology, and give you errors about the mdp file.

Reply:
Ok, I like the idea that mimicpy can fix the topology.